Microsoft.Toolkit.Services/Core/IStorageManager.cs

```diff
@@ -14,13 +16,14 @@ public interface IStorageManager
         /// </summary>
         /// <param name="key"> Token value </param>
         /// <returns> Returns a string value</returns>
-        string Get(string key);
+        Task<string> Get(string key);
```

Comment:
+Breaking Change

Reply:
This should be renamed to GetAsync. All methods that return task should have the postfix "Async". I forgot to mention that before, sorry.